application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -246,6 +248,37 @@
       $('#downloadFilesModal .download-logs-date-fields').hide();
     }
   });
+
+  // Currently, the Dashboard gadgets title doesn't have an ID set (https://jira.xwiki.org/browse/XWIKI-20600). Solution
+  // is to check for an empty ID and remove the said title as it does not point anywhere in the page. Also, this
+  // function will check for an empty name (TOC macro will generate a title with the name of the page if the
+  // header is empty; this behaviour is modified by https://github.com/xwiki/xwiki-rendering/pull/253/).
+  const removeEmptyHeaders = function(element) {
```

Comment:
Maybe `removeBrokenHeaders`? Since there is more than one case that is covered by this method